conda-forge.yml

```diff
@@ -1,7 +1,8 @@
 appveyor:
   secure: {BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1}
-compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
 travis:
   secure: {BINSTAR_TOKEN: YDNSAcI5rPI5IMOnqr+V4QJt1PfOmgurq6koFLW6OJjaawtEVEDIrL3qcvemey4vlY/qXdGAchS2aXuN2fh/YJRlJAS684zYfcSSoq5Lz7tvKFJnCsOv7WEoVEySdcCzX7HfDTuLU+tjHUpXdyBip6lEP5Fp5UvtB7ES/okkX2k2hXjvWELiefIsER3hE/lAqsnP7o5276Jcm9sz8Gj0CLNdI1wUWNZkE0DkCsRNe6G8JBfeVWRiCYyyYE3MNdx5jl9kTYofxUmBikc1PisbY1D0dWinmZeCkcVP9C8NlYzuohG1p9VYlLR6xYnhcCYvQarjNcQOKW+MFcXwH83aJ2gEub3MjtczbyvVxzyCKcsLqvyJRzjBDbVl1kDRTBrmeBHXjgYieb80GpFtU79TWie/aJMsbLkz9KWqKFkp2aADy46Tm5VcQ37oV3ZrELWSUzSbZcvI7KEYzIJ+7iTuXOfJ2CZhrsh5LFMmIsVRKRxHUkt3xZ0TatOP+fY/7z2n+V7CXEcMrzg5NQe3uqb9H3yb4fr95aJkl76VJ9xYddz6jk8Ax1ZGlicIqDqyRkL3o3936cSKD7ljH2RjyLqjC6Z8saeRP9w3aM+nx3i0TppDeP9jeLBC5SlLanvDpq3/Wr+xGjpSUZc6TRFjhquz3hEP2zzFdNqCwOh5Jfzm/P0=}
+provider:
+  linux_aarch64: azure
```

Comment:
Any reason not to make this `default`?

What about adding `ppc64le` as well?

Reply:
maybe that works also ... I know that the easiest is that aarch64 builds like his "Intel brother" on the standard docker anvil and the Linux-Intel CI defaults to azure, that's why I immediately state azure :-) 

I don't have experience with PowePC, and what's more is that I haven't got such a machine, so I can't evaluate correct behavior. 

I thus think it is better to wait for someone who needs it. He/she then adds the support and can verify correct behavior :-)

Reply:
Please make this `default` as Azure is emulated and `default` is native.

In conda-forge we have been adding both of these at the same time. It would be good to do the same thing here. Could you please add it?

Here are the relevant docs: https://conda-forge.org/docs/maintainer/conda_forge_yml.html#provider

Reply:
Sure, let's see how that works :-)
one question though ... the doc states : "default to enable a platform and choose an appropriate CI" where exactly do I chose the CI then ?